src/components/BooksContainer/AddBook.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../styles/addbook.css';
+
+export default function AddBook() {
+  return (
+    <form action="#">
+      <h2 className="adding-span">Add a new book</h2>
+      <div className="adding-form d-flex">
+        <input className="book-input" />
+        <select className="selector">
+          <option selected disabled>Categories</option>
+        </select>
+        <button type="button">ADD BOOK</button>
+      </div>
+    </form>
```

Comment:
- [ ] This component should render a form with inputs for both the title and author of the book. You're currently having only the title input. Add the author input to your form.

Reply:
@mke2111 Thank you very much for your kind feedback. Can you take a look at this : 
![image](https://user-images.githubusercontent.com/83329154/140003408-0f6de618-5a07-4463-93c4-109642f01d32.png)

In the sneak-peek task we're supposed to build something that follows that design. I myself thought about how can we add an author but then I thought maybe the input is going to be sent to an API and then retrieve the whole information about the book including that. Should I add an input still or should I keep it like that?

Reply:
That can be confusing if you focus on the final product of the project. But for this milestone's requirements, the author's input should be present. This will later be removed while designing the final display. For now, do not worry about the final display.

Reply:
@mke2111 Alright then I will add the input field for the author as well right away and I'll give them placeholders to be presentable.